src/app/components/mnoe-api/product-instances.svc.coffee

```diff
@@ -7,24 +7,9 @@ angular.module 'mnoEnterpriseAngular'
 
     productInstancesPromise = null
```

Comment:
I think we can remove this now that we aren't cacheing.

Reply:
Ah yes, good catch 👍

Reply:
Adding the cache back in as the workaround implemented no longer relies on the product instances themselves